{"uid":"171803f8fc5cd9fa","name":"Update full name succesfully","fullName":"Update full name succesfully","historyId":"Update full name succesfully","time":{"start":1677618681636,"stop":1677618681641,"duration":5},"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/35bc8fc69ed971ff26706ee1eb7ca697/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/35bc8fc69ed971ff26706ee1eb7ca697/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/runner/work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 161\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","flaky":false,"newFailed":false,"newBroken":true,"newPassed":false,"retriesCount":0,"retriesStatusChange":false,"beforeStages":[{"name":"BeforeFeature [10000]","time":{"start":1677618681628,"stop":1677618681635,"duration":7},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateProject [1]","time":{"start":1677618681636,"stop":1677618681636,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CreateAnItem [2]","time":{"start":1677618681636,"stop":1677618681636,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"BeforeScenario [10000]","time":{"start":1677618681636,"stop":1677618681636,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"testStage":{"status":"broken","statusMessage":"An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/35bc8fc69ed971ff26706ee1eb7ca697/screenshot. The exception message was: Connection refused (localhost:4444) -> Connection refused (localhost:4444) -> Connection refused","statusTrace":"OpenQA.Selenium.WebDriverException: An unknown exception was encountered sending an HTTP request to the remote WebDriver server for URL http://localhost:4444/session/35bc8fc69ed971ff26706ee1eb7ca697/screenshot. The exception message was: Connection refused (localhost:4444)\n ---> System.Net.Http.HttpRequestException: Connection refused (localhost:4444)\n ---> System.Net.Sockets.SocketException (111): Connection refused\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.ThrowException(SocketError error, CancellationToken cancellationToken)\n   at System.Net.Sockets.Socket.AwaitableSocketAsyncEventArgs.System.Threading.Tasks.Sources.IValueTaskSource.GetResult(Int16 token)\n   at System.Net.Sockets.Socket.<ConnectAsync>g__WaitForConnectWithCancellation|277_0(AwaitableSocketAsyncEventArgs saea, ValueTask connectTask, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   --- End of inner exception stack trace ---\n   at System.Net.Http.HttpConnectionPool.ConnectToTcpHostAsync(String host, Int32 port, HttpRequestMessage initialRequest, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.ConnectAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.CreateHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.AddHttp11ConnectionAsync(HttpRequestMessage request)\n   at System.Threading.Tasks.TaskCompletionSourceWithCancellation`1.WaitWithCancellationAsync(CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.GetHttp11ConnectionAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpConnectionPool.SendWithVersionDetectionAndRetryAsync(HttpRequestMessage request, Boolean async, Boolean doRequestAuth, CancellationToken cancellationToken)\n   at System.Net.Http.RedirectHandler.SendAsync(HttpRequestMessage request, Boolean async, CancellationToken cancellationToken)\n   at System.Net.Http.HttpClient.<SendAsync>g__Core|83_0(HttpRequestMessage request, HttpCompletionOption completionOption, CancellationTokenSource cts, Boolean disposeCts, CancellationTokenSource pendingRequestsCts, CancellationToken originalCancellationToken)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.MakeHttpRequest(HttpRequestInfo requestInfo)\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   --- End of inner exception stack trace ---\n   at OpenQA.Selenium.Remote.HttpCommandExecutor.Execute(Command commandToExecute)\n   at OpenQA.Selenium.WebDriver.Execute(String driverCommandToExecute, Dictionary`2 parameters)\n   at OpenQA.Selenium.WebDriver.GetScreenshot()\n   at SeleniumTest.Tests.Hooks.Hooks.CaptureScreenshot() in /home/runner/work/GUI-Framework/GUI-Framework/3.Tests/UI/Hooks/Hooks.cs:line 161\n   at TechTalk.SpecFlow.Bindings.BindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)\n   at Allure.SpecFlowPlugin.AllureBindingInvoker.InvokeBinding(IBinding binding, IContextManager contextManager, Object[] arguments, ITestTracer testTracer, TimeSpan& duration)","steps":[{"name":"Given  the user is logged in","time":{"start":1677618681636,"stop":1677618681638,"duration":2},"status":"failed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"attachments":[],"parameters":[],"stepsCount":1,"attachmentsCount":0,"shouldDisplayMessage":true,"hasContent":true},"afterStages":[{"name":"UpdateFullName [10000]","time":{"start":1677618681641,"stop":1677618681867,"duration":226},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"CaptureScreenshot [10000]","time":{"start":1677618681867,"stop":1677618681868,"duration":1},"status":"broken","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false},{"name":"AfterFeature [10000]","time":{"start":1677618681870,"stop":1677618681870,"duration":0},"status":"passed","steps":[],"attachments":[],"parameters":[],"stepsCount":0,"attachmentsCount":0,"shouldDisplayMessage":false,"hasContent":false}],"labels":[{"name":"thread","value":"NonParallelWorker"},{"name":"host","value":"fv-az283-835"},{"name":"feature","value":"Full name update"},{"name":"tag","value":"Smoke"},{"name":"tag","value":"Regression"},{"name":"tag","value":"restore.default.fullname"},{"name":"resultFormat","value":"allure2"}],"parameters":[],"links":[],"hidden":false,"retry":false,"extra":{"severity":"normal","retries":[],"categories":[{"name":"Test defects","matchedStatuses":[],"flaky":false}],"history":{"statistic":{"failed":0,"broken":8,"skipped":0,"passed":32,"unknown":0,"total":40},"items":[{"uid":"e503cf0a327c98d6","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/670//#testresult/e503cf0a327c98d6","status":"passed","time":{"start":1677618136152,"stop":1677618140495,"duration":4343}},{"uid":"61ae10af98c34bc4","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/669//#testresult/61ae10af98c34bc4","status":"passed","time":{"start":1677617204969,"stop":1677617209079,"duration":4110}},{"uid":"ef83318710b90355","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/669//#testresult/ef83318710b90355","status":"passed","time":{"start":1677616369275,"stop":1677616374469,"duration":5194}},{"uid":"86a778feb4f34341","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/668//#testresult/86a778feb4f34341","status":"passed","time":{"start":1677616387606,"stop":1677616393026,"duration":5420}},{"uid":"8b9109839e3660ee","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/668//#testresult/8b9109839e3660ee","status":"passed","time":{"start":1677615984782,"stop":1677615988950,"duration":4168}},{"uid":"e16b28a9ee9c3fdf","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/667//#testresult/e16b28a9ee9c3fdf","status":"passed","time":{"start":1677614569661,"stop":1677614573908,"duration":4247}},{"uid":"b801fe4b11a2e8f3","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/666//#testresult/b801fe4b11a2e8f3","status":"passed","time":{"start":1677613827814,"stop":1677613833177,"duration":5363}},{"uid":"55a1ae40b9495ecc","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/664//#testresult/55a1ae40b9495ecc","status":"passed","time":{"start":1677613532127,"stop":1677613536526,"duration":4399}},{"uid":"91dd071529910344","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/665//#testresult/91dd071529910344","status":"passed","time":{"start":1677613262786,"stop":1677613267163,"duration":4377}},{"uid":"75187607c4c2f2ab","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/664//#testresult/75187607c4c2f2ab","status":"passed","time":{"start":1677612884699,"stop":1677612889068,"duration":4369}},{"uid":"afb40eb18e0197da","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/662//#testresult/afb40eb18e0197da","status":"passed","time":{"start":1677612386221,"stop":1677612391125,"duration":4904}},{"uid":"adc1373ac0d67d61","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/662//#testresult/adc1373ac0d67d61","status":"passed","time":{"start":1677611736442,"stop":1677611740240,"duration":3798}},{"uid":"d1e129874dc4271","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/661//#testresult/d1e129874dc4271","status":"passed","time":{"start":1677611142797,"stop":1677611147199,"duration":4402}},{"uid":"eb1b52c91bf69baf","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/659//#testresult/eb1b52c91bf69baf","status":"passed","time":{"start":1677610397889,"stop":1677610402272,"duration":4383}},{"uid":"a701088a58e60245","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/659//#testresult/a701088a58e60245","status":"passed","time":{"start":1677609968488,"stop":1677609972428,"duration":3940}},{"uid":"a2e2bfcb701f33d6","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/658//#testresult/a2e2bfcb701f33d6","status":"passed","time":{"start":1677609473730,"stop":1677609478256,"duration":4526}},{"uid":"12edd8a8d7374a24","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/656//#testresult/12edd8a8d7374a24","status":"passed","time":{"start":1677608802538,"stop":1677608807670,"duration":5132}},{"uid":"ccb6eaa70db67149","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/657//#testresult/ccb6eaa70db67149","status":"broken","statusDetails":"Exception has been thrown by the target of an invocation. -> The type initializer for 'Todoly.Core.Helpers.ConfigModel' threw an exception. -> Key not found : BROWSERSTACK-USERNAME","time":{"start":1677608789913,"stop":1677608789914,"duration":1}},{"uid":"4a54254e8ae10507","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/4//#testresult/4a54254e8ae10507","status":"passed","time":{"start":1677608153235,"stop":1677608156852,"duration":3617}},{"uid":"2611e53084056455","reportUrl":"https://BC-LT-02.github.io/GUI-Framework/655//#testresult/2611e53084056455","status":"passed","time":{"start":1677607709761,"stop":1677607714741,"duration":4980}}]},"tags":["restore.default.fullname","Regression","Smoke"]},"source":"171803f8fc5cd9fa.json","parameterValues":[]}